paddle/fluid/distributed/collective/process_group_nccl.cc

```diff
@@ -93,8 +106,17 @@ ProcessGroupNCCL::ProcessGroupNCCL(
     const std::shared_ptr<phi::distributed::Store>& store,
     int rank,
     int size,
-    int gid)
-    : ProcessGroupWithStream(rank, size, gid), store_(store) {}
+    int gid,
+    int64_t timeout)
+    : ProcessGroupWithStream(rank, size, gid),
+      store_(store),
+      pg_timeout_(timeout) {
+  // TODO(yuwentao01): for debug
+  char* test_hang = std::getenv("TEST_HANG");
```

Comment:
Change the name to  FLAGS_xx

Reply:
> 

it will remove, only temporarily used for manual manufacturing hang to verifiy trace function